### ISSUE TITLE
Add `X-Frame-Options: DENY` header to prevent clickjacking

### ISSUE THREAD

**Issue by lyoshenka** [LBRYFoundation/lbryorg#16]
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options

**Comment by iamuditpatel** [comment]
@lyoshenka thanks for opening this Issue.

@Eniamza  i try to fix this above problem but no luck due to my limited knowledge of jekyll but here i find something useful that can help you to fix this problem as you implemented jekyll in this website you get better understanding of jekyll and how it's works.

https://github.com/mattcone/markdown-guide
this above link website use jekyll and it's hosted on github pages like we do and this website already have correct way to add Add `X-Frame-Options: DENY` header to prevent clickjacking.

![bug1](https://user-images.githubusercontent.com/11346292/99532999-2b826d80-29cb-11eb-85ba-c843d9831014.PNG)


but i not touching the base setting of jekyll that you set because if i am not sure what i doing it's good i don't touch the code as fix the problem by luck it's very risky way to solve any problem.

**Comment by Eniamza** [comment]
I'll be looking into it the next week.

**Comment by Eniamza** [comment]
@lyoshenka A quick research shows that github forcefully implants xframe: deny in all page. Either It's top level page or not

There for both iframe and clickjacking isn’t possible as for writing

Thanks for pointing out!

**Comment by lyoshenka** [comment]
I don't think that's true. Look at this https://securityheaders.com/?q=https%3A%2F%2Flbry.org&followRedirects=on

**Comment by lyoshenka** [comment]
It says we're missing a whole bunch of security headers, including `X-Frame-Options`

**Comment by Coolguy3289** [comment]
I noticed that lbry.com is covered it seems. Do you know how you were able to add these options for that site? I did some googling and there doesn't seem to be a cut and dry way from what I was reading.

**Comment by lyoshenka** [comment]
is this running on Github Pages?

lbry.com is not a jekyll site. its php, running on a server. it may be that github pages doesn't let you add these kinds of headers.

**Comment by Coolguy3289** [comment]
Yes, It's a GH Pages site. Since lbry.com is on Github, I presumed it was GitHub Pages. I don't think we have the ability to add the headers

**Comment by Eniamza** [comment]
Yes,  lbry.org builds as gh pages. Though github doesn’t allow setting security headers

We may follow this :     https://www.rzegocki.pl/blog/custom-http-headers-with-github-pages/

**Comment by Eniamza** [comment]
If anyone can, Implant these headers, Feel free to contribute!
